ingest-app/docs/api.md

```diff
@@ -1192,6 +1197,64 @@ Ingest permissions for granule subscriptions are granted through the provider vi
 For lack of a better ACL, ingest permissions for collection subscription are granted through the SYSTEM OBJECT TAG_GROUP ACL update permission.
 
 %GENERIC-DOCS%
+
+## <a name="publish-drafts"></a> Publish Drafts 
+### <a name="publish-non-variable-draft"></a> Publish Non Variable Draft 
+#### <a name="publish-non-variable-draft-endpoint"></a> /publish/&lt;non-variable-draft-concept-id&gt;/&lt;native-id&gt;
+
+Non variable drafts can be published, i.e. ingested into CMR as non variable concept, through the non variable draft publishing endpoint.
+
+Example: Publish non variable draft OOD1200000005-PROV1 using native-id orderoption1. Note: if Content-Type is passed in, use it, otherwise use the format stored in the database for OOD1200000005-PROV1.  
+
+```
+curl -XPUT \
+     -H "Authorization:  XXXX" \ 
+     -H "Content-Type:application/vnd.nasa.cmr.umm+json" \
+     %CMR-ENDPOINT%/publish/OOD1200000005-PROV1/orderoption1
+```
+#### Successful Response in XML
+
+```
+<?xml version="1.0" encoding="UTF-8"?><result>
+    <concept-id>OO1200000006-PROV1</concept-id>
+    <revision-id>1</revision-id>
+    <warnings></warnings>
+    <existing-errors></existing-errors>
+</result>
+
+```
+
+### <a name="publish-variable-draft"></a> Publish Variable Draft 
+#### <a name="publish-variable-draft-endpoint"></a> /collections/&lt;collection-concept-id&gt;/&lt;collection-revision-id&gt;/variables/&lt;native-id&gt;/publish/&lt;variable-draft-concept-id&gt; 
+
+Variable drafts can be published, i.e. ingested into CMR as variable concept, through the variable draft publishing endpoint.
+
+Example: Publish variable draft VD1200000008-PROV1 using native-id var1. Note: if Content-Type is passed in, use it, otherwise use the format stored in the database for VD1200000008-PROV1. 
```

Comment:
Change the text to state:
Example: Publish a variable draft record with the variable draft concept id of VD1200000008-PROV1 using a new native-id of var1. Note: The CMR will use the HTTP header Content-Type if it is passed in to describe the to be published records format. Otherwise the CMR will assume the format of the new record to be published is the same as the draft record.